utilities/create_site.sh

```diff
@@ -15,16 +15,21 @@ fi
 git clone --branch gh-pages --single-branch https://github.com/GoogleCloudPlatform/google-cloud-java/ tmp_gh-pages
 mkdir -p tmp_gh-pages/$SITE_VERSION_BASE
 mvn site -DskipTests -Djava.util.logging.config.file=logging.properties
-mvn site:stage --quiet -Djava.util.logging.config.file=logging.properties -DtopSiteURL=http://googlecloudplatform.github.io/google-cloud-java/site/${SITE_VERSION_BASE}/
-cd tmp_gh-pages
 
-for dir in ../target/staging/${SITE_VERSION_BASE}*
+version_bases=(${SITE_VERSION_BASE} latest)
+rm -rf tmp_gh-pages/latest
```

Comment:
Sorry, I missed this initially. How will this behave together with the corresponding "git add latest" for cases when there is already something in tmp_gh-pages/latest in github? I hope git is smart enough to remove what is not needed and update what was updated (no other commands needed, like explicit "git rm")?